src/main/java/net/silthus/ebean/EbeanWrapperBungee.java

```diff
@@ -0,0 +1,13 @@
+package net.silthus.ebean;
+
+import net.md_5.bungee.api.plugin.Plugin;
+
+/**
```

Comment:
Please remove the unneeded comment. Source control will show the author and date.